### DISCUSSION TITLE
No 'install for my user only' in MacOS

### DISCUSSION THREAD

**Discussion by Paulie420** [Xaymar/obs-stream-effects#962] [General]
### Discussion Thread

StreamFX

### Provide a summary of the Improvement

Am I missing something? I've installed OBS w/ home-brew so its in ~/Library/Application Support/obs-studio - even when option-clicking StreamFX.pkg, there is no 'install for my user only' option - and I can't manually move the folder to obs as it doesn't see it.

Any newbie suggestions? Sorry to tie up a 'issue'.

**Comment by Xaymar** [comment]
On MacOS, the Installer automatically chooses between System-wide and User-mode installations and you do not need to do anything else. The [Installation guide](https://github.com/Xaymar/obs-StreamFX/wiki/Installation) should be followed to the letter, without adding any weird steps not described in it.

Up until StreamFX 0.12.0, the Installer would only allow System-wide installation on MacOS. With StreamFX 0.12.0 and beyond, it is now possible to pick between the two, though the Installer will try its best to ensure you install as a User-only installations, due to OBS Studio 28.x no longer supporting System-wide installations.

**Reply by Paulie420** [reply]
Thanks a ton - once I grabbed the 0.12.0 version StreamFX installed on the first go. Thanks for replying to a derpy question. :P

**Comment by jarno527** [comment]
I'm trying to install on Mac OS 13.1 and OBS 28.0.3 (64 bit on a M1). I've spent about 30 minutes and can't get it to go. I followed the installation guide laboriously, installing the latest production build ([streamfx-macos-10.15-0.11.1.0-g81a96998.pkg](https://github.com/Xaymar/obs-StreamFX/releases/download/0.11.1/streamfx-macos-10.15-0.11.1.0-g81a96998.pkg)). It goes through the install dialogue and states a successful install, but nothing appears in the plug ins folder. 

I'd be glad to use this plug in if I can but I'm stuck. Thanks for any help anyone can give!

**Reply by jarno527** [reply]
Ok. I just installed the alpha (which I was trying to avoid) and it went great. That's my answer. Thanks!

**Comment by reinermatthew** [comment]
I'm trying to install streamfx-macos-10.15-0.11.1.0-g81a96998.pkg and it's also not installing the app for the local user, just for all users. I tried just copying the StreamFX folder to the local user plugins folder, but still, none of the filters appear in OBS.
Has anyone found a solution, other than installing the alpha? I'm really looking to use the stable build.

**Reply by Xaymar** [reply]
Only 0.12 and beyond support the new requirements set by obs
